src/Schema/Builder.php

```diff
@@ -165,9 +178,42 @@ public function getTables($schema = null)
             ];
         }
 
-        usort($collections, function ($a, $b) {
-            return $a['name'] <=> $b['name'];
-        });
+        usort($collections, fn ($a, $b) => $a['name'] <=> $b['name']);
+
+        return $collections;
+    }
+
+    /** @param  string|null $schema Database name */
+    public function getViews($schema = null)
+    {
+        $db = $this->connection->getDatabase($schema);
+        $collections = [];
+
+        foreach ($db->listCollections() as $collectionInfo) {
+            $collectionName = $collectionInfo->getName();
+
+            // Skip system collections
```

Comment:
Is this necessary? I wouldn't expect any system collection to have a "view" type, so the conditional below should already filter these out.